src/effects/effectchain.cpp

```diff
@@ -340,8 +340,7 @@ void EffectChain::slotControlChainPresetSelector(double value) {
 }
 
 void EffectChain::slotControlLoadedChainPresetRequest(double value) {
-    // subtract 1 to make the ControlObject 1-indexed like other ControlObjects
-    int index = static_cast<int>(value) - 1;
+    int index = static_cast<int>(value);
```

Comment:
Why not add `QuickEffectChain::slotControlLoadedChainPresetRequest(double value)` so the index is stll 1-based for standard chains and 0-based for QuickEffects?

We just need to add a note to the documentation https://github.com/mixxxdj/manual/issues/501